tests/Resources/Page/ODH/JupyterHub/AccessGroups.resource

```diff
@@ -0,0 +1,161 @@
+*** Settings ***
+Resource      ../../../Page/Components/Components.resource
+Resource      ../../../Page/OCPDashboard/UserManagement/Groups.robot
+Resource      ../../../Common.robot
+Resource      JupyterHubSpawner.robot
+Resource      HighAvailability.robot
+Library       JupyterLibrary
+Library       OpenShiftCLI
+
+
+*** Variables ***
+${CUSTOM_ADMINS_GROUP}=   custom-admins-group
+${CUSTOM_USERS_GROUP}=    custom-users-group
+${STANDARD_ADMINS_GROUP}=  rhods-admins
+${STANDARD_USERS_GROUP}=   rhods-users
+
+
+*** Keywords ***
+Create Custom Groups
+    [Documentation]    Creates two user groups: custom-admins and customer-users
+    ${admin_created}=    Run Keyword And Return Status    Create Group  group_name=${CUSTOM_ADMINS_GROUP}
+    IF    $admin_created == False
+          FAIL  Creation of ${CUSTOM_ADMINS_GROUP} group failed. Check the logs
+    END
+    ${user_created}=     Run Keyword And Return Status    Create Group  group_name=${CUSTOM_USERS_GROUP}
+    IF    $user_created == False
+        FAIL    Creation of ${CUSTOM_USERS_GROUP} group failed. Check the logs
+    END
+    Navigate To Page    User Management    Groups
+    Wait Until Page Contains  Create Group
+    Capture Page Screenshot    user_groups.png
+    Page Should Contain    ${CUSTOM_ADMINS_GROUP}
+    Page Should Contain    ${CUSTOM_USERS_GROUP}
+
+
+Delete Custom Groups
+    [Documentation]    Deletes two user groups: custom-admins and customer-users
+    Delete Group  group_name=${CUSTOM_ADMINS_GROUP}
+    Delete Group  group_name=${CUSTOM_USERS_GROUP}
+    Navigate To Page    User Management    Groups
+    Wait Until Page Contains  Create Group
+    Capture Page Screenshot    user_groups_deletion.png
+    Page Should Not Contain    ${CUSTOM_ADMINS_GROUP}
+    Page Should Not Contain    ${CUSTOM_USERS_GROUP}
+
+Add Test Users To Custom Groups
+    [Documentation]    Adds two tests users to custom-admins and customer-users groups
```

Comment:
This keyword seems very specific for a test case. If not, at least username should be a parameter I think

Reply:
all the kw in this file are very specific for a test case. I've put all of them together to have a clear Test case file because there are many of them